### ISSUE TITLE
Xcode 14 - unable to compile SDK for generic platform

### ISSUE THREAD

**Issue by pandel** [jitsi/jitsi-meet#12168]
Hi!

I am unable to compile the current SDK with Xcode 14. When i start

```
xcodebuild archive \
    -workspace ios/jitsi-meet.xcworkspace \
    -scheme JitsiMeetSDK  \
    -configuration Release \
    -sdk iphoneos \
    -destination='generic/platform=iOS' \
    -archivePath ios/sdk/out/ios-device \
    VALID_ARCHS=arm64 \
    ENABLE_BITCODE=NO \
    SKIP_INSTALL=NO \
    BUILD_LIBRARY_FOR_DISTRIBUTION=YES
```

I get

```
note: Building targets in dependency order
/Users/holgerpandel/Source/jitsi-meet/ios/Pods/Pods.xcodeproj: error: Signing for "React-Core-AccessibilityResources" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'React-Core-AccessibilityResources' from project 'Pods')
warning: Run script build phase 'Copy sounds' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'JitsiMeetSDK' from project 'sdk')
warning: Run script build phase 'Package React bundle' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'JitsiMeetSDK' from project 'sdk')
/Users/holgerpandel/Source/jitsi-meet/ios/Pods/Pods.xcodeproj: error: Signing for "GoogleSignIn-GoogleSignIn" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'GoogleSignIn-GoogleSignIn' from project 'Pods')
** ARCHIVE FAILED **
```

I am able to compile for the simulator without problems. What can I do?

Regards,
Holger

**Comment by saghul** [comment]
Have you selected a team for signing, as the error suggests?

**Comment by pandel** [comment]
I tried to set the team ID via `DEVELOPMENT_TEAM` variable on the command line, but the lead to the next error:
`No signing certificate "iOS Development" found: No "iOS Development" signing certificate matching team ID XY`

I am not that deep into iOS development. It worked up to now without any problems. I was always able to compile the SDK following your SDK docs, copy both frameworks into my project, compile my app and send it to the App Store. I tried to find out what this "iOS Development" cert is and why my normal development cert isn't working here, but I really don't get it right now...

BTW, during `pod install` I ran into a problem with this: https://github.com/facebook/react-native/pull/33967/files

**Comment by saghul** [comment]
Sounds like a setup issue. Are you lees in with your Apple ID on Xcode?

**Comment by pandel** [comment]
I succeeded now. I reset everything including Xcode cache, profiles, whatnot and again, added the DEVELOPMENT_TEAM variable on the command line. That did the trick. It would be nice if you'd give a hint to this on the iOS SDK docs page. And perhaps also that Giphy Framework has to be added now. I had to add this to my app, too, to make it work again but did not see any information in the docs regarding Giphy.

Many thanks!
Holger
